compiler/rustc_span/src/source_map.rs

```diff
@@ -462,6 +462,17 @@ impl SourceMap {
         )
     }
 
+    pub fn span_to_locatio_info(&self, sp: Span) -> (Symbol, usize, usize, usize, usize) {
```

Comment:
*location

Reply:
fixed.

ps: pretty sure responded to this. not sure why my comments got reset.